package.json

```diff
@@ -28,6 +28,8 @@
         "test:chrome": "node tools/build.js normalize & karma start --chrome",
         "test:firefox": "node tools/build.js normalize & karma start --firefox",
         "test:debug": "node tools/build.js normalize & karma start --no-single-run --chrome",
+        "test:fast": "node tools/build.js normalize & karma start karma.fast.conf.js --chrome",

```

Comment:
I think for all test commands, we only need to keep the following:

* test, test:chrome: test using chrome with original config so it will do type check
* test:firefox: test using firefox with original config so it will do type check
* test:debug: test using chrome with new config, skip type check  **(Modified)**
* test:debug-firefox: test using firefox with new config, skip type check  **(Modified)**
* test:coverage: test and add coverage report

So no no need to introduce new command
When test with debugging, always using fast config
The original test/test:chrome/test:firefox command is not used very much.